src/modules/unicode/unicode.cpp

```diff
@@ -231,6 +230,7 @@ Unicode::Unicode(Instance *instance)
 Unicode::~Unicode() {}
 
 void Unicode::trigger(InputContext *inputContext) {
+    if (!data_.load()) return;
```

Comment:
I suggest to make trigger() has a bool return value so it can preserve the old behavior. Though it's unlikely to happen because the data file is not really intended to be override by others.